### ISSUE TITLE
Users would like to be able to copy studies

### ISSUE THREAD

**Issue by adswafford** [biocore/qiita#2207]
Maybe a button/link on the create new study page that says "Create from existing study"

**Comment by antgonza** [comment]
Interesting, what will be the goal of this?

**Comment by adswafford** [comment]
I can't recall what the original request we got was to do this, but I can think of two use cases:
1) There is a public study that I want to work with, but I want to change/collapse/group/etc. the metadata in a different way for an analysis; Rather than downloading the metadata, prep, fastq etc. and uploading everything fresh and processing again, I want to just copy the study and edit the metadata; Maybe this can be handled better by allowing users to modify and upload the metadata associated with an analysis?
2) I've tried to process my files and completely messed something up and I can't get everything deleted because there are blocking files somewhere and/or I'm confused. I don't want to have to download and upload all the files again, so I just want to duplicate a study and then delete down to the point where I can restart. Although in this case, you'd want the study to just copy the files (metadata, preps, and raw data) w/o associating them I guess?
